KomodoPlatform/atomicDEX-Desktop
# Get icons from coins repo at build time

closes https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/2037
To test:
- load app
- confirm all coin logos are present

Note: until https://github.com/KomodoPlatform/coins/pull/610 is merged, the icons in addressbook for network standard address input will not be present.
